docs/supported-services/ecs.rst

```diff
@@ -53,6 +53,16 @@ Parameters
      - No
      - 
      - If your task needs routing from a load balancer, this section can be used to configure the load balancer's options.
+   * - logging
+     - string
+     - No
+     - "enabled"
+     - Turns CloudWatch logging on or off. Must be either "enabled" or "disabled". See :ref:`ecs-logging` for more.
+   * - log_retention_in_days
+     - number
+     - No
+     -
```

Comment:
Can you put the default value for log retention here in this column?

Reply:
Yeah, that had just occurred to me.  I've now got it defaulting to '0', which means 'indefinite'.